src/types/api/schema/replication.ts

```diff
@@ -0,0 +1,94 @@
+import type {IssueMessage} from '../query';
+
+import type {TPathID} from './shared';
+
+/**
+ * source: https://github.com/ydb-platform/ydb/blob/main/ydb/core/protos/replication.proto
+ */
+interface TStaticCredentials {
+    User?: string;
+    Password?: string;
+    PasswordSecretName?: string;
+}
+
+/**
+ * source: https://github.com/ydb-platform/ydb/blob/main/ydb/core/protos/replication.proto
+ */
+interface TOAuthToken {
+    Token?: string;
+    TokenSecretName?: string;
+}
+
+/**
+ * source: https://github.com/ydb-platform/ydb/blob/main/ydb/core/protos/replication.proto
+ */
```

Comment:
There is no need to copy paste source for every type. Such comment is needed to be able to easily find protos and update types, when source files have changed